test/unit/test_check_usernames.py

```diff
@@ -1,24 +1,29 @@
+import configparser
+
 import pytest
 import vcr
 
 from fal.controllers.check_usernames import check
 
+config = configparser.ConfigParser()
+config.read("config.ini")
+
 
-@vcr.use_cassette('test/unit/vcr_cassettes/check_usernames/check-username-success.yaml')
+@vcr.use_cassette(f"{config['vcr']['path']}/check_usernames/check-username-success.yaml")
```

Comment:
I think it would look cleaner if we stored the vcr path into a global variable and used it instead of reading from config each time.

Reply:
Isn't the config file already a type of global variable? What would you suggest?

Reply:
I just meant like at the top of the file we do 
```python
vcr_path = config['vcr']['path']
```
And in the decorator
```python
f"{vcr_path}/check_usernames/..."

It's not a huge difference but it's a little easier to read imo

Reply:
ah gotcha. can do